runtime/Cargo.toml

```diff
@@ -10,10 +10,12 @@ edition = "2018"
 
 [dependencies]
 bincode = "1.3.1"
+blake3 = "0.3.6"
 bv = { version = "0.11.1", features = ["serde"] }
 byteorder = "1.3.4"
 bzip2 = "0.3.3"
 crossbeam-channel = "0.4"
+digest = "0.9.0"
```

Comment:
`blake3` re-exports `digest` https://docs.rs/blake3/0.3.6/blake3/traits/index.html.  Better not to try to keep these in sync